src/methods/geom_relations/intersects.jl

```diff
@@ -269,7 +201,7 @@ line segments, line strings, linear rings, polygons, and multipolygons. If no
 intersection points were possible given geometry extents, return nothing. If
 none are found, return an empty list.
 """
-function intersection_points(::GI.AbstractTrait, a, ::GI.AbstractTrait, b)
+function intersection_points(::GI.AbstractGeometryTrait, a, ::GI.AbstractGeometryTrait, b)
```

Comment:
This should let `FeatureTrait` work too...
```suggestion
function intersection_points(::GI.AbstractTrait, a, ::GI.AbstractTrait, b)
```